TypeCobol/Compiler/Nodes/Node.cs

```diff
@@ -848,6 +849,7 @@ public DataDefinition GetDataDefinitionFromStorageAreaDictionary(StorageArea sea
         
         public DataDefinition GetDataDefinitionForQualifiedName(QualifiedName qualifiedName, bool? isReadDictionary=null)
```

Comment:
Same here:
Can you add attribute [CanBeNull] before QualifiedName qualifiedName ?
This way Resharper will know that the value can be null.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1595/commits/59139c835bd4751fb47886a0d3305206f1272b62